src/nvim/highlight_group.c

```diff
@@ -363,6 +363,8 @@ static const char *highlight_init_light[] = {
   "DiffText             guifg=NvimDarkGrey1  guibg=NvimLightCyan             ctermfg=15 ctermbg=6",
   "Directory            guifg=NvimDarkCyan                                   ctermfg=6",
   "ErrorMsg             guifg=NvimDarkRed                                    ctermfg=1",
+  "StderrMsg            guifg=NONE           guibg=NONE                      ctermfg=NONE ctermbg=NONE",
```

Comment:
StderrMsg should be distinguishable somehow, maybe make it the same as the "Warning" hl. @echasnovski  ?